### ISSUE TITLE
AVFS: How to use it in protable?

### ISSUE THREAD

**Issue by emako** [vapoursynth/vapoursynth#400]
When I want to try avfs.
>avfs template.vpy -d
ERROR: Unable to open PFM interface. PFM probably not installed.
There are not protable avfs in protable vapoursynth.

I looked at 'vsinstaller.iss'.
May I can install PFM by this command?
>.\pismo\pfm-191-vapoursynth-win.exe install

But pfm-191-vapoursynth-win.exe was not found in protable vapoursynth.

How can I use the protable avfs but not using vsinstaller?

**Comment by myrsloik** [comment]
Yes, you have to install the PFM runtime. The reason it isn't included is because it's by nature not very portable since it requires installation. Maybe I'll include it in the package just for convenience in R44.

**Comment by emako** [comment]
Thanks for your help.
Avfs is very useful for previewing!
